rotkehlchen/db/accounting_rules.py

```diff
@@ -204,6 +205,19 @@ def add_linked_setting(
                 f'Link of rule for {rule_property} to setting {setting_name} already exists',
             ) from e
 
+    def missing_accounting_rule(
```

Comment:
Hmm that's what I meant but not for a single event. As you end up doing multiple reads in a for loop.

Instead pass a list of events in this funciton and return them a list of true/false for missing rules.